app/build.gradle

```diff
@@ -97,6 +99,7 @@ android {
             buildConfigField "boolean", "ENABLE_ENGELSYSTEM_SHIFTS", "true"
             buildConfigField "String", "TRACE_DROID_EMAIL_ADDRESS", '"tobias.preuss+36c3@googlemail.com"'
             buildConfigField "String", "SCHEDULE_FEEDBACK_URL", '"https://frab.cccv.de/en/36C3/public/events/%s/feedback/new"'
+            buildConfigField "boolean", "ENABLE_CHAOSFLIX_EXPORT", "true"
```

Comment:
Please remove this redundant setting from the `ccc36c3` flavor since it is enabled by default.

Reply:
Resolved in 5147a5af7b566465f39d8b1c4b3c483cdf9416f9. Thank you.